parsers/test/mocks/hrote.json

```diff
@@ -3,1202 +3,1202 @@
     {
       "ID": "portfolio",
       "Timestamp": "2019-11-03 16:00:00+01:00",
```

Comment:
Could you actually try to replace the content of this file with [this JSON](https://files.hrote.hr/files/EKO_BG/FORECAST/SOLAR/FTP/TEST_DRIVE/21.5.2020.json)?

My thinking is that it would be good to also update the timestamps since the old timestamps point at the time when the values used to be strings.

Reply:
Thanks for reviewing! I replaced the contents of that file. 

After that I found that for the tests to work I also needed to update dates in the tests and a couple of the other mock files to 21/5/2020 to match that file, and to update two of the assertions in the first test to match the content of the new JSON.